src/components/LocalReader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Papa from 'papaparse';
+
+function LocalReader({ setData }: { setData: (data: any[]) => void }) {
```

Comment:
Might want to make this name more explicit to include CSV and the local refers to a local file reader, in-case we need multiple readers.